js/app.js

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
General comment -- in the future please be sure to do this if it asks:  (from the assignment)
2. Copy the questions that you are answering into your file (and comment it out) and write the answer below the question.